can/broadcastmanager.py

```diff
@@ -293,6 +293,10 @@ def _run(self) -> None:
         msg_index = 0
         msg_due_time_ns = time.perf_counter_ns()
 
+        if USE_WINDOWS_EVENTS:
+            # Make sure the timer is non-signaled before entering the loop
+            win32event.WaitForSingleObject(self.event.handle, 0)
```

Comment:
Maybe we could call SetWaitableTimer here instead of in `start()`?

Reply:
Sure that should be possible, in another PR maybe?

Reply:
I meant SetWaitableTimer instead of WaitForSingleObject. Wouldn't that solve this issue?

Reply:
This would give us essentially what the current code with the bug is...

Reply:
Apparently the handle is signaled immediately, if the duetime parameter is 0.

Can you try this?
```python3
    def start(self) -> None:
        self.stopped = False
        if self.thread is None or not self.thread.is_alive():
            name = f"Cyclic send task for 0x{self.messages[0].arbitration_id:X}"
            self.thread = threading.Thread(target=self._run, name=name)
            self.thread.daemon = True

            if USE_WINDOWS_EVENTS:
                win32event.SetWaitableTimer(
                    self.event.handle,
                    -1 * self.period_ns // 100,
                    self.period_ms,
                    None,
                    None,
                    False,
                )

            self.thread.start()
```